engine/android/app/build.gradle

```diff
@@ -31,12 +24,25 @@ android {
         }
     }
 	
+	signingConfigs {
+    release {
+		//////////// SIGNING FOR CUSTOM APK ////////////////
+		keyAlias "alias_name"
+		keyPassword "password"
+		storeFile file("C:/openbor/engine/android/my-key.jks")
```

Comment:
Please allow me to chime in, how can we make this portable for build on other platforms?

Reply:
I can only support windows for now.  If someone wants to compile on other OS they will have to work how to do it based on my guide.  Maybe you could help write a Linux specific tutorial ?

Reply:
for the record, updated this on https://github.com/DCurrent/openbor/pull/144